### PR TITLE
Fix target selector for imported projects

### PR DESCRIPTION
**Pull request by bryanhiggins**

### CODE REVIEW

**Comment by bryanhiggins** [comment]
Closes #212

File: lib/ui/views/targetSelector.js [hunk 1]
```diff
@@ -26,13 +26,10 @@ TargetSelectorView = Backbone.View.extend({
...
         // Use the template to create the html to display
         var view = this,
-            showDevices = view.model.get("buildSettings").device,
+            buildSettings = view.model.get("buildSettings"),
+            showDevices = !buildSettings || typeof buildSettings.device === "undefined" || buildSettings.device,
             template;
 
-        if (typeof showDevices === "undefined") {
-            showDevices = true;
-        }
```
Comment (rcruz) [inline lib/ui/views/targetSelector.js:34]: What the reasoning behind removing this piece?

Comment (bryanhiggins) [inline lib/ui/views/targetSelector.js:34]: Because the same logic is above

Comment (rcruz) [inline lib/ui/views/targetSelector.js:34]: I see it now.

**Comment by rcruz** [comment]
r+

**Pull request merged**